torrentfileQt/menu.py

```diff
@@ -23,129 +23,172 @@
-        self.actionDarkTheme = QAction(self.window)
         self.actionLightTheme = QAction(self.window)
-        self.actionAddProfile = QAction(self.window)
+        self.actionDarkTheme = QAction(self.window)
+        self.actionFontPlus = QAction(self.window)
+        self.actionFontMinus = QAction(self.window)
+        self.addAction(self.actionExit)
+        self.addAction(self.actionFontPlus)
+        self.addAction(self.actionFontMinus)
+        self.addAction(self.actionLightTheme)
+        self.addAction(self.actionDarkTheme)
+        self.actionFontPlus.setText("Font +")
+        self.actionFontMinus.setText("Font -")
         self.actionDarkTheme.setText("Dark Theme")
         self.actionLightTheme.setText("Light Theme")
-        self.actionRepo.setText("Github Repository")
         self.actionExit.setText("Exit")
-        self.actionAbout.setText("About")
-        self.actionDocs.setText("Documentation")
-        self.actionAddProfile.setText("Add Profile")
-        self.add_profile_actions()
-        self.file_menu.addAction(self.actionExit)
-        self.file_menu.addAction(self.actionLightTheme)
-        self.file_menu.addAction(self.actionDarkTheme)
-        self.help_menu.addAction(self.actionAbout)
-        self.help_menu.addAction(self.actionDocs)
-        self.help_menu.addAction(self.actionRepo)
-        self.profile_menu.addAction(self.actionAddProfile)
+        self.actionFontPlus.triggered.connect(self.increaseFont)
+        self.actionFontMinus.triggered.connect(self.decreaseFont)
         self.actionExit.triggered.connect(self.exit_app)
-        self.actionAbout.triggered.connect(self.about_qt)
-        self.actionDocs.triggered.connect(documentation)
-        self.actionRepo.triggered.connect(repository)
-        self.actionAddProfile.triggered.connect(self.add_profile)
         self.actionLightTheme.triggered.connect(self.light_theme)
         self.actionDarkTheme.triggered.connect(self.dark_theme)
+        self.actionFontPlus.setObjectName("actionIncreaseFont")
+        self.actionFontMinus.setObjectName("actionDecreaseFont")
+        self.actionExit.setObjectName("actionExit")
         self.actionLightTheme.setObjectName("actionLightTheme")
         self.actionDarkTheme.setObjectName("actiondDarkTheme")
-        self.actionDocs.setObjectName("actionDocs")
-        self.actionAddProfile.setObjectName("actionAddProfile")
-        self.actionExit.setObjectName("actionExit")
-        self.actionAbout.setObjectName("actionAbout")
 
-    def light_theme(self):
+    def exit_app(self):
+        """Close application."""
+        self.parent().app.quit()  # pragma: nocover
+
+    @staticmethod
+    def light_theme():
         """Change the GUI theme for the application."""
-        self.window.setStyleSheet(light_theme)
+        QApplication.instance().styleManager.setTheme("light")
 
-    def dark_theme(self):
+    @staticmethod
+    def dark_theme():
         """Change the GUI application to dark theme."""
```

Comment:
changed them back to instance methods since I am now using the `self` argument in the method body and no longer relying on the QApplication class method for grabbing the app instance.